java/clients/src/test/java/sleeper/clients/testutil/WiremockEMRTestHelper.java

```diff
@@ -97,31 +101,28 @@ public static ResponseDefinitionBuilder aResponseWithNumRunningClusters(int numR
     }
 
      public static ResponseDefinitionBuilder aResponseWithNumRunningApplications(int numRunningApplications) {
-        return aResponseWithNumRunningApplications(numRunningApplications, false);
+        List<ApplicationState> states = new ArrayList<>();
+        for (int i = 0; i < numRunningApplications; i++) {
+            states.add(ApplicationState.STARTED);
+        }
+        return aResponseWithNumRunningApplications(states);
      }
 
-    public static ResponseDefinitionBuilder aResponseWithNumRunningApplications(int numRunningApplications, boolean includeStoppedState) {
+    public static ResponseDefinitionBuilder aResponseWithNumRunningApplications(List<ApplicationState> states) {
         StringBuilder applicationBody = new StringBuilder("{\"applications\": [");
-        for (int i = 1; i <= numRunningApplications; i++) {
-            String state = "RUNNING";
-
-            if (i == 1 && includeStoppedState) {
-                state = "STOPPED";
-            }
-
+        for (int i = 1; i < states.size(); i++) {
```

Comment:
This will skip the first state in the test. Everywhere that uses this passes in either one state or no states. This means it will always generate a response with no applications.

It looks like the reason the tests are passing is because none of them assert whether jobs were queried, whether jobs were cancelled, or whether the application was stopped. Please could you add those assertions?